orca-echo/src/main/java/com/netflix/spinnaker/orca/echo/util/ManualJudgmentAuthzGroupsUtil.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 OpsMx, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.echo.util;
+
+import com.netflix.spinnaker.fiat.model.Authorization;
+import java.util.List;
+import java.util.Map;
+
+public class ManualJudgmentAuthzGroupsUtil {
+
+  public static boolean checkAuthorizedGroups(
+      List<String> userRoles, List<String> stageRoles, Map<String, Object> permissions) {
```

Comment:
Expected to see a test for this class.

Reply:
6 test cases are written/added in ManualJudgmentStageSpec.groovy only to test this logic. Please have a look at this test case.

@Unroll
  void "should return execution status based on authorizedGroups"() {

Reply:
Seems weird to have a test case for a different class (ManualJudgmentStage) specifically testing a different class, doesn't it?

Reply:
We are using the logic in the ManualJudgmentStage.groovy. So i have written 6 test cases to check the  ManualJudgmentAuthzGroupsUtil.java logic in the ManualJudgmentStageSpec.groovy.

Reply:
You're also using it in a controller, which means that the test cases don't belong where there are today.

I'd say the tests that are testing this code should live in `ManualJudgmentAuthzGroupsUtilSpec`, then test the implementation in `ManualJudgmentStageSpec` and `OperationsControllerSpec`, as you would with any other unit under test.

Reply:
done.